docs/metrics.md

```diff
@@ -327,6 +327,21 @@ A gauge of the number of queue items that have not been processed yet.
 
 See [queue adds count](#queue_adds_count) for details.
 
+#### `version`
+
+Build metadata for this Controller.
+
+| attribute        | explanation                                                                                           |
+|------------------|-------------------------------------------------------------------------------------------------------|
+| `version`        | The version of Argo                                                                                   |
+| `platform`       | The [Go platform](https://go.dev/doc/install/source#environment) compiled for. Example: `linux/amd64` |
+| `goversion`      | Version of Go used                                                                                    |
+| `build_date`     | Build date for this workflow controller                                                               |
+| `compiler`       | The compiler used to build this workflow controller e.g. `gc`                                         |
```

Comment:
```suggestion
| `build_date`     | Build date                                                               |
| `compiler`       | The compiler used. Example: `gc`                                         |
```

You missed these two [previous](https://github.com/argoproj/argo-workflows/pull/13269#discussion_r1661377037) [suggestions](https://github.com/argoproj/argo-workflows/pull/13269#discussion_r1661391935). GH collapsed those two, so I'm guessing you just didn't see it